packages/pds-ember/addon/components/pds/button/docs.mdx

```diff
@@ -1,10 +1,10 @@
-import { Props } from '@storybook/addon-docs/blocks';
+import { ArgsTable } from '@storybook/addon-docs/blocks';
```

Comment:
FYI: Storybook 6 has deprecated the use of the `<Props>` component in favor of the `<ArgsTable>` component.